Template/LO/SubProcesses/myamp.f

```diff
@@ -52,6 +52,7 @@ logical function cut_bw(p)
       logical             OnBW(-nexternal:0)     !Set if event is on B.W.
       common/to_BWEvents/ OnBW
       
+      include 'vector.inc'
       include 'coupl.inc'
```

Comment:
same

Reply:
same as above, vector.inc is needed by coupl.inc